.github/workflows/rockspec-info.yml

```diff
@@ -6,24 +6,24 @@ on:
         description: "JSON object with rockspec info"
         value: ${{ jobs.rockspec-info.outputs.info }}
 
```

Comment:
Having release-please update this file was a round-about way of obtaining the current package versions. It's better to just grab it out of `.release-please-manifest` - that's what it is for anyways.